### ISSUE TITLE
rejigger breaks sometimes

### ISSUE THREAD

**Issue by sapid** [browndar/scheduler#1]
Just making a ticket so I remember what the issue was

[rejigger](https://github.com/browndar/scheduler/blob/8fe8c64/src/main/java/org/sptc/Groups.java#L30) function breaks if the group population isn't large enough to contain a certain number of groups (e.g. 13 people participating)

**Comment by sapid** [comment]
A solution I shared. A problem with this solution is that it trends toward groups we consider too small.

```
import random

rsvps = 7
ids = ([i for i in range(rsvps)])
pods = []

random.shuffle(ids)

if len(ids) < 6:
  print("Just one group this month.")

while len(ids) % 4 != 0:
  pods.append([ids.pop(), ids.pop(), ids.pop()])

while len(ids) != 0:
  pods.append([ids.pop(), ids.pop(), ids.pop(), ids.pop()])

print(pods)
```

**Comment by browndar** [comment]
Great! I implemented your idea real quick in commit 2908feaea29a021d3267dcb2394706f0db90018b, Not really sure how to connect these two in github. Think I can close this one with the commit.
